pkg/ranchhand/rke.go

```diff
@@ -36,14 +39,45 @@ services:
     snapshot: true
     creation: 6h
     retention: 24h
+  kube-api:
+    pod_security_policy: true
+    extra_args:
+      anonymous-auth: "false"
+      profiling: "false"
+      repair-malformed-updates: "false"
+      service-account-lookup: "true"
+      enable-admission-plugins: "ServiceAccount,NamespaceLifecycle,LimitRanger,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota,DefaultTolerationSeconds,AlwaysPullImages,DenyEscalatingExec,NodeRestriction,EventRateLimit,PodSecurityPolicy"
+      admission-control-config-file: "/etc/kubernetes/admission.yaml"
```

Comment:
the location of the hardcoded files might be worth a constant to be shared between this file and `rke.go`